doc/schema.graphql

```diff
@@ -427,51 +434,50 @@ type MutationApicluster {
     """
     sched_move_quota: Long
   ): VshardGroup!
-  auth_params(
-    """
-    The `Long` scalar type represents non-fractional signed whole numeric
-    values. Long can represent values from -(2^52) to 2^52 - 1, inclusive.
-    """
-    cookie_max_age: Long
 
-    """The `Boolean` scalar type represents `true` or `false`."""
-    enabled: Boolean
+  """Restart replication on specified by uuid servers"""
+  restart_replication(uuids: [String!]): Boolean
```

Comment:
```suggestion
  """Restart replication on specified by uuid servers"""
  restart_replication(uuids: [String!]): Boolean
```
Unfortunately I can't get what you're saying. I suppose there is the following problem: A participle cannot stand in front of the noun like it often does in Russian.